tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2158,20 +2160,17 @@ std::unique_ptr<Thunk> IrEmitterUnnested::BuildConditionalThunk(
   TF_CHECK_OK(CheckConditionalBuffersShareAllocation(
       hlo, ir_emitter_context_->buffer_assignment()));
 
-  HloComputation* true_computation = hlo->true_computation();
-  IrEmitterUnnested ir_emitter_true(
-      hlo_module_config_, true_computation,
-      ir_emitter_context_,
-      GetTargetIRBuilder().GetTargetMachineFeatures());
-  TF_CHECK_OK(true_computation->Accept(&ir_emitter_true));
-
-  HloComputation* false_computation = hlo->false_computation();
-  IrEmitterUnnested ir_emitter_false(
-         hlo_module_config_, 
-         false_computation,
-         ir_emitter_context_,
-          GetTargetIRBuilder().GetTargetMachineFeatures());
-  TF_CHECK_OK(false_computation->Accept(&ir_emitter_false));
+  std::vector<BufferAllocation::Slice> branch_operands;
```

Comment:
@sumesh13 I think the updated logic which stores into `branch_thunks` is the new logic, right?